src/ioOperations.ts

```diff
@@ -0,0 +1,16 @@
+import {WS_EVENTS} from "./events";
+import {DbInterface} from "./database/db";
+
+const db = new DbInterface();
+
+export function sendQuestionToPlayers(roomId: string, io, questions, questionIndex) {
+
+  db.getQuestion(questions[questionIndex]).then((question) => {
+    const data = {
+      question: question?.Question,
+      answers: Array.from(question?.PossibleAnswers.values())
+    }
+
+    io.to(roomId).emit(WS_EVENTS.NEW_QUESTION, data);
```

Comment:
Rather than importing `io` in here, i think it might be better to use this function to generate the data to be emitted and return it when this function is called. Thoughts?

Reply:
yeah, sounds good, I refactored it so all the core lives outside of `io`.